### ISSUE TITLE
linux-image-extra not available on AWS server

### ISSUE THREAD

**Issue by joshbrw** [puppetlabs/puppetlabs-docker#38]
Error:

```
Error: Execution of '/usr/bin/apt-get -q -y -o DPkg::Options::=--force-confold install linux-image-extra-4.4.0-1039-aws' returned 100: Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package linux-image-extra-4.4.0-1039-aws
E: Couldn't find any package by glob 'linux-image-extra-4.4.0-1039-aws'
E: Couldn't find any package by regex 'linux-image-extra-4.4.0-1039-aws'
Error: /Stage[main]/Docker::Install/Package[linux-image-extra-4.4.0-1039-aws]/ensure: change from purged to present failed: Execution of '/usr/bin/apt-get -q -y -o DPkg::Options::=--force-confold install linux-image-extra-4.4.0-1039-aws' returned 100: Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package linux-image-extra-4.4.0-1039-aws
E: Couldn't find any package by glob 'linux-image-extra-4.4.0-1039-aws'
E: Couldn't find any package by regex 'linux-image-extra-4.4.0-1039-aws'
```

I've temporarily patched this by setting `$manage_kernel => false`, but seems to be an issue.

**Comment by davejrt** [comment]
Apologies for the delay in getting back to you. We'll look into fixing the logic for this in an upcoming release.

**Comment by davideagle** [comment]
Any news on this bug, or a workaround other than $manage_kernel => false ?

**Comment by LongLiveCHIEF** [comment]
@davejrt do we even need the `manage_kernel` switch anymore? I believe at this point it's only needed for  a combination of docker and operating system that I think this module would no longer even support.

**Comment by LongLiveCHIEF** [comment]
> There is a ticket in our backlog to remove anything that is no longer relevant to the operating system we no longer support after the fork.

Assuming the supported opertating systems shown in `metadata.json` is correct, then my PR #102 will take care of that.

**Comment by davejrt** [comment]
Fixed in PR #116
